src/oscillator.js

```diff
@@ -627,4 +626,4 @@ p5.SqrOsc = function (freq) {
 
 p5.SqrOsc.prototype = Object.create(p5.Oscillator.prototype);
 
-export default p5.Oscillator;
```

Comment:
why this change?

Not opposed to it but I do think we should be consistent about using names exports (like this one) or default exports?